WordPress/gutenberg
# Stylebook: render overview colors in 4 columns

## What?
Stylebook: render overview colors in 4 columns

## Why?
Closes: https://github.com/WordPress/gutenberg/issues/67592

## How?
Add a columns prop to ColorExamples component. Add specific styles for color grids with 4 colors per row.

## Testing Instructions
- Navigate to the stylebook.
- Observe the colors in the overview page, they should be rendered in 4 columns.
- Navigate to the Colors tab of the style book.
- Observe the colors, they should be rendered in 2 columns.

## Screenshots or screencast 

|Before|After|
|-|-|
|![Screenshot from 2024-12-04 15-28-06](https://github.com/user-attachments/assets/5e900752-6975-4a37-bb5b-f3f64e89fcd5) ![Screenshot from 2024-12-04 15-27-53](https://github.com/user-attachments/assets/fdf4f282-f8bf-4b6d-8aa1-afdea9b55e10)|![Screenshot from 2024-12-04 15-18-06](https://github.com/user-attachments/assets/8fe97902-33aa-439b-bf05-deac72a96cea) ![Screenshot from 2024-12-04 15-17-22](https://github.com/user-attachments/assets/ddff3825-ce41-4bca-b7c7-90cfdc6aa4ce) |
